receiver/sqlserverreceiver/README.md

```diff
@@ -68,6 +78,22 @@ Example with named instance:
 
 The full list of settings exposed for this receiver are documented in [config.go](./config.go) with detailed sample configurations in [testdata/config.yaml](./testdata/config.yaml).
 
+Top query collection enabled:
+```yaml
+    receivers:
+      sqlserver:
+        collection_interval: 5s
+        username: sa
+        password: securepassword
+        server: 0.0.0.0
+        port: 1433
+        
+        lookback_window: 60
+        max_sample_query_count: 1000
+        top_query_count: 200
+        logs:
+          enable_top_query_collection: true
+```
```

Comment:
Could we add documentation for what attribute we are going to push into logs? Just like the metrics documentation? Otherwise, our users can only check the code to know the data type and data unit.

This documentation should have the attribute name, unit, and value type.

Reply:
updated